### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by calchen**
here is my issue #4574

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
         }
     ],
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^5.6 || >=7.0.8",
```
Comment (kubawerlos) [inline composer.json]:
```suggestion
        "php": "^5.6 || ^7.0.8",
```

**Comment by SpacePossum** [comment]
Closing following the explaining of Julien. This project does not require PHP 7.0.8; the console component does. Composer will figure this out for you if you install on a PHP supported by both.

**Comment by calchen** [comment]
> The current constraint already allows _any_ PHP 7 version, including yours (7.0.0). Not only will this change not fix your issue, it will add a new constraint to prevent your specific version (`>=7.0.8` and `^7.0.8` both reject version `7.0.0`).
> 
> Allowing PHP 7.0.0 would require to change the constraint in `symfony/console`, but if they claim they need PHP to be at least 7.0.8, they probably have a good reason.
> 
> I don't think there is much to do here. I'd suggest you update your PHP version.

thanks for your explaining, i will update my php version

**Comment by calchen** [comment]
@julienfalque thanks for your explaining, i will update my php version

**Pull request closed without merging**